src/DotnetDocument/Syntax/SyntaxUtils.cs

```diff
@@ -110,6 +110,22 @@ public static IEnumerable<string> ExtractBaseTypes(ClassDeclarationSyntax classD
             return new List<string>();
         }
 
+        /// <summary>
+        /// Gets the type.
+        /// </summary>
+        /// <param name="constructorDeclarationSyntax">The constructor declaration syntax.</param>
+        /// <returns></returns>
+        public static string GetType(ConstructorDeclarationSyntax constructorDeclarationSyntax)
```

Comment:
Could we name this to something more explicit like `GetClassName` or `ExtractClassName`? :pray:

Reply:
Fix